ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -321,6 +325,12 @@ export default defineComponent({
           ? 'Organization'
           : 'Owner'
       }),
+      ownerOrOrgHeader: computed((): string => {
+        const found = getManufacturedHomeSearchResults.value.results
+        if (found) {
+          return found[0]?.organizationName ? 'Organization' : 'Owner'
+        } else return ''
+      }),
```

Comment:
This does seem very similar to `personOrOrgLabel`. 
Do we NEED both computed values?
Can we get away with one?

Reply:
I found that personOrOrgLabel never returned 'Organization' and was planning to remove it, I guess I forgot whoops!

Reply:
Sounds good, thanks!